v2/manifests/core-ons/dp-frontend-feedback-controller.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  dp-frontend-feedback-controller:
+    x-repo-url: "https://github.com/ONSdigital/dp-frontend-feedback-controller"
+    build:
+      context: ${DP_REPO_DIR:-../../../..}/dp-frontend-feedback-controller
+      dockerfile: Dockerfile.local
+    command:
+      - reflex
+      - -d
+      - none
+      - -c
+      - ./reflex
```

Comment:
feedback controller doesn't look to have a `reflex` file.

Reply:
Added to PR for docker file